### DISCUSSION TITLE
自动文献计数及编码

### DISCUSSION THREAD

**Discussion by klocklockloc** [windingwind/zotero-actions-tags#348] [Ideas]
从Endnote转移到Zotero后，唯一怀念的功能是Endnote在添加文献时会给一个唯一的“RecordNumver”。有没有可能让Tag帮助实现此功能，比如每当我添加一个新item时，检查目前所有item中“档案”的数值，并记下最大值，自动为当前导入的item的“档案”写入新数值，新数值时最大值加1？

**Comment by windingwind** [comment]
不清楚有什么实际用途，每个item有独立id和key，但是对用户不显示。如果只是为了计数有多少条目，不选中任何条目时右侧即显示总数

**Comment by klocklockloc** [comment]
感谢老大回复！
我知道如何显示条目总数和任一分类的总数。用简短且唯一的数字编号对应到文献，这对我来说有两个作用：1. 方便在一段时间记忆（比如写某篇论文的时候临时记忆或标记，这回非常方便，只需写下几个数字）；2. 方便对论文按添加时间排序、查找（通常临近添加的论文具有相关性）。
我刚从Endnote转到Zotero，Endnote是自带的，Zotero我还不太熟练，似乎是不能实现。我目前的做法是在“档案”一栏手动编号，在列中添加“档案”，这么干效率很低。
我搜过国外的论坛，有不少人也希望有这个功能。
祝生活愉快！
---- 回复的原邮件 ----
***@***.***>发送日期2024年7月23日 ***@***.***>***@***.***>,
***@***.***>主题Re: [windingwind/zotero-actions-tags] 自动文献计数及编码 (Discussion #348)
不清楚有什么实际用途，每个item有独立id和key，但是对用户不显示。如果只是为了计数有多少条目，不选中任何条目时右侧即显示总数

**Reply by windingwind** [reply]
按添加时间排序，在库中新增一列添加时间即可。相对而言，记忆抽象的数字，肯定不如记忆文献关键词来得容易。一篇论文几十上百个引用，我相信不太可能全部以数字记忆。且z的添加引文窗口会显示已引用的文献，一般敲一两个字母即可找到了，我感觉独立id实用性其实不高
